Add support of pre-release Python versions (#112)
actions/setup-python: README.md

> 

Reviewer: There is a `What's new` section where support for pre-release versions can be mentioned: https://github.com/actions/setup-python#whats-new

I would suggest an extra bullet point that says: `Support for pre-release versions of Python`

Author: Thanks!

```

> Download and set up the accurate pre-release version of Python:

Reviewer: ```suggestion
Download and set up a pre-release version of Python:
```

Author: Thanks!

```yaml
steps:
- uses: actions/checkout@v2
- uses: actions/setup-python@v2
  with:
    python-version: '3.9.0-beta.4'
- run: python my_script.py
```

> Download and set up the latest available version of Python (including both pre-release and stable versions):

Reviewer: ```suggestion
Download and set up the latest available version of Python (includes both pre-release and stable versions):
```

Author: Thanks!
